src/styles/pages/get-involved/donate.module.scss

```diff
@@ -0,0 +1,51 @@
+.sectionContainer {
+  display: flex;
+  padding-top: 32px;
+  background-color: var(--color-white);
+  font-family: var(--font-inter);
+  padding: var(--edge-spacer);
+}
+
+.donateBox {
+  margin-top: 16px;
+  background-color: var(--color-beige);
+  padding-top: 16px;
+  padding-bottom: 16px;
+  padding-left: 32px;
+  padding-right: 32px;
+  border-radius: 5px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+  font-size: 1.25rem;
+}
+
+.donateHeading {
+  font-size: 2rem;
+  text-align: center;
+  color: var(--color-maroon);
+  padding-bottom: 16px;
+}
+
+.startSubtext1 {
+  padding-bottom: 16px;
+}
+
+.centeredSubtext {
```

Comment:
you kinda don't need any of these stylings if you just change the text-align of cneteredTextBox to center. The less stylings you have, the easier it is to debug and make your design responsive otherwise changing one thing might throw off a lot of other stuff.